### DISCUSSION TITLE
Wifi On/Off no beeps Mini 11

### DISCUSSION THREAD

**Discussion by audetra** [gopro/labs#387] [General]
Just updated to latest version. Tried all my current QR codes, they worked great and getting the beeps until I got to the Wi-Fi on or off with the beeps set to 70% with all other settings set to off.  Tried again with fresh using QR Control app, still no beeps. Checked with the mini settings and noticed it set it to low, but still was not hearing anything.  Was nice to hear that confirmation with a beep, rather having to look at the mini screen for confirmation.

**Comment by dnewman-gpsw** [comment]
There does seem to must less beeping with the current Labs build on Mini.  There is a hack of a work around, in QRControl add `!B2` to the Added Command, field.  !B2 is the beep/blink twice command.

I have added the missing beeps to the bug list.

**Reply by audetra** [reply]
Thanks dnewman-gpsw ! 
I had wondered if there was a command for this. All working great with the added command.
